scripts/generate_docs.sh

```diff
@@ -16,21 +15,24 @@
 set -e
 
 my_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-root_path="$my_path/.."
+declare -r root_path="$my_path/.."
 version=$(git describe --abbrev=0 --tags || echo "0.0.0")
-modules=(CoreMetrics Metrics)
+declare -r modules=(CoreBaggage)
+main_module="CoreBaggage"
+declare -r project_xcodeproj_name="swift-distributed-tracing-baggage-core"
+declare -r project_gh_name="swift-distributed-tracing-baggage-core"
 
 if [[ "$(uname -s)" == "Linux" ]]; then
   # build code if required
-  if [[ ! -d "$root_path/.build/x86_64-unknown-linux" ]]; then
+  if [[ ! -d "$root_path/.build/x86_64-unknown-linux-gnu" ]]; then
```

Comment:
this is when running ` docker-compose -f docker/docker-compose.yaml -f docker/docker-compose.1804.53.yaml run shell` it seems ubuntu is now doing the `-gnu`